lib/module.js

```diff
@@ -74,6 +74,11 @@ function Module(id, parent) {
 }
 module.exports = Module;
 
+const builtinModules = Object.keys(process.binding('natives'))
+  .filter((s) => !s.includes('/'));
+
```

Comment:
The reason I'm interested in this is because I have need in user-land as well, but providing incorrect information means I'll ignore this exposed property as bogus. I don't see how exposing something that's not accurate is a good thing.

Reply:
I believe the filter should be `!s.startsWith("internal/")`

Reply:
Which aligns with

https://github.com/nodejs/node/blob/07a4fa3dbefdc7f740a5a281f713998e71aab8f3/lib/internal/bootstrap_node.js#L517

https://github.com/nodejs/node/blob/07a4fa3dbefdc7f740a5a281f713998e71aab8f3/lib/internal/bootstrap_node.js#L577-L583

Reply:
You could even use `NativeModule.isInternal` as the filter.

Reply:
```js
const builtinModules = Object.keys(NativeModule._source)
  .filter(NativeModule.isInternal)
```

Reply:
erm... that is `NativeModule.nonInternalExists`